wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -0,0 +1,791 @@
+| `da.md` | Danish | Dansk |
+| `de.md` | German | Deutsch |
+| `gr.md` | Greek | Ελληνικά |
+| `es.md` | Spanish | Español |
+| `fi.md` | Finnish | Suomi |
+| `fr.md` | French | Français |
+| `hu.md` | Hungarian | Magyar |
+| `id.md` | Indonesian | Bahasa Indonesia |
+| `it.md` | Italian | Italiano |
+| `ja.md` | Japanese | 日本語 |
+| `ko.md` | Korean | 한국어 |
+| `nl.md` | Dutch | Nederlands |
+| `no.md` | Norwegian | Norsk |
+| `pl.md` | Polish | Polski |
+| `pt.md` | Portuguese | Português |
+| `pt-br.md` | Brazilian Portuguese | Português (Brasil) |
+| `ro.md` | Romanian | Română |
+| `ru.md` | Russian | Русский |
+| `sk.md` | Slovak | Slovenčina |
+| `sv.md` | Swedish | Svenska |
+| `th.md` | Thai | ไทย |
+| `tr.md` | Turkish | Türkçe |
+| `uk.md` | Ukrainian | Українська мова |
+| `vi.md` | Vietnamese | Tiếng Việt |
+| `zh.md` | Chinese (Simplified) | 简体中文 |
+| `zh-tw.md` | Traditional Chinese (Taiwan) | 繁體中文（台灣） |
+
+*Note: The website will give readers their selected language's version of an article. If it is not available, the English version will be given.*
+
+If a language is not listed above, append `?locale={langcode}` to the URL (e.g. `https://osu.ppy.sh/help/wiki/Welcome?locale=zh`). You can also help translate the [osu-web via Crowdin](https://crowdin.com/project/osu-web). If you need help with translating or want to discuss translating decisions, discuss in the [osu!dev Discord](https://discord.gg/ppy) (`#osu-web` channel).
+
+### Content parity
+
+Translations are subject to strict content parity with their English article, in the sense that they must have the same message, regardless of grammar and syntax. Any changes to the translations' meanings must be accompanied by equivalent changes to the English article.
+
+There are some cases where the content is allowed to differ:
+
+- Articles originally written in a language other than English (in this case, English should act as the translation)
+- Explanations of English words that are common terms in the osu! community
+- External links
+- Tags
+
+## Front matter
+
+Front matter must be placed at the very top of the file. It is written in [YAML](https://en.wikipedia.org/wiki/YAML#Example "YAML Wikipedia article") and describes additional information about the article. This must be surrounded by three hyphens (`---`) on the lines above and below it, and an empty line must follow it before the title heading.
+
+### Articles that need help
+
+The `needs_cleanup` tag may be added to articles that need rewriting or formatting help. It is also acceptable to open an issue on GitHub for this purpose. This tag must be written as shown below:
+
+```yaml
+needs_cleanup: true
+```
+
+When adding this tag to an article, [comments](#comments) should also be added to explain what needs to be done to remove the tag.
+
+*Notice to translators: Avoid translating English articles with this tag. In addition to this, this tag should be added when the translation needs its own clean up.*
```

Comment:
is it a notice or a note? also, please put it at the beginning of the sub-subsection